XML/TestCases/FunctionalTests-InfiniBand.xml

```diff
@@ -157,6 +157,79 @@
 		<Area>INFINIBAND</Area>
 		<Tags>rdma</Tags>
 	</test>
+	<!-- HPC-X MPI -->
+	<test>
+		<testName>INFINIBAND-HPCX-MPI-2VM</testName>
+		<testScript>VERIFY-INFINIBAND-MultiVM.ps1</testScript>
+		<setupType>TwoVM1Dep</setupType>
+		<OverrideVMSize>Standard_HB60rs</OverrideVMSize>
+		<SubtestValues>SOME,TEXTS,NEEDS,TO,BE,PRESENT,HERE,FOR,PRINTING,TEST,SUMMARY</SubtestValues>
+		<files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\TestRDMA_MultiVM.sh,.\Testscripts\Linux\SetupRDMA.sh</files>
+		<Priority>1</Priority>
+		<TestParameters>
+			<param>num_reboot="INFINIBAND_HPCX_TOTAL_REBOOT_COUNT"</param>
+			<param>mpi_settings="INFINIBAND_HPCX_MPI_SETTINGS_IB"</param>
+			<param>ib_nic="INFINIBAND_IB_NIC"</param>
+			<param>imb_mpi1_tests="INFINIBAND_HPCX_MPI_TESTNAMES"</param>
+			<param>mpi1_ppn=INFINIBAND_HPCX_PPN</param>
+			<param>imb_mpi1_tests_iterations=INFINIBAND_MPI_TEST_COUNT</param>
+			<param>imb_rma_tests="INFINIBAND_HPCX_RMA_TESTNAMES"</param>
+			<param>rma_ppn=INFINIBAND_HPCX_PPN</param>
+			<param>imb_rma_tests_iterations=INFINIBAND_HPCX_RMA_TEST_COUNT</param>
+			<param>imb_nbc_tests="INFINIBAND_HPCX_NBC_TESTNAMES"</param>
+			<param>nbc_ppn=INFINIBAND_HPCX_PPN</param>
+			<param>imb_nbc_tests_iterations=INFINIBAND_HPCX_NBC_TEST_COUNT</param>
+			<param>p2p_ppn=INFINIBAND_HPCX_PPN</param>
+			<param>imb_p2p_tests_iterations=INFINIBAND_HPCX_P2P_TEST_COUNT</param>
+			<param>io_ppn=INFINIBAND_HPCX_PPN</param>
+			<param>imb_io_tests_iterations=INFINIBAND_HPCX_IO_TEST_COUNT</param>
+			<param>mpi_type="hpcx"</param>
+			<param>mlx_ofed_partial_link=MLX_OFED_PARTIAL_LINK</param>
+			<param>hpcx_mpi=HPCX_PARTIAL_LINK</param>
+			<param>intel_mpi_benchmark=INTEL_MPI_BENCHMARK_DOWNLOAD</param>
+			<param>walaagent_repo=WALAAgent_REPO</param>
+		</TestParameters>
+		<Platform>Azure</Platform>
+		<Category>Functional</Category>
+		<Area>INFINIBAND</Area>
+		<Tags>rdma</Tags>
+	</test>
+	<test>
```

Comment:
We did not run 32VM but add it here?

Reply:
I did a run of 32VM over Centos 7.6 HPC:

[LISAv2 Test Results Summary]
Test Run On           : 07/15/2019 07:13:03
ARM Image Under Test  : OpenLogic : CentOS-HPC : 7.6 : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:5:47

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 INFINIBAND           INFINIBAND-HPCX-MPI-32VM                                                          PASS               339.69
        InfiniBand-Verification-1-FirstBoot : eth0 IP : PASS
        InfiniBand-Verification-1-FirstBoot : IBV_PINGPONG : PASS
        InfiniBand-Verification-1-FirstBoot : PingPong Intranode : PASS
        InfiniBand-Verification-1-FirstBoot : IMB-MPI1 : PASS
        InfiniBand-Verification-1-FirstBoot : IMB-P2P : PASS
        InfiniBand-Verification-1-FirstBoot : IMB-NBC : PASS
        InfiniBand-Verification-2-Reboot : eth0 IP : PASS
        InfiniBand-Verification-2-Reboot : IBV_PINGPONG : PASS
        InfiniBand-Verification-2-Reboot : PingPong Intranode : PASS
        InfiniBand-Verification-2-Reboot : IMB-MPI1 : PASS
        InfiniBand-Verification-2-Reboot : IMB-P2P : PASS
        InfiniBand-Verification-2-Reboot : IMB-NBC : PASS

Reply:
Oh! It's great. I was wondering because I did not see the log or OneNote pages.